### PR TITLE
add pulsar exporter to otel-contrib distributions

### PR DESCRIPTION
**Pull request by tjiuming**
add pulsar exporter to otel-contrib distributions

### CODE REVIEW

**Comment by tjiuming** [comment]
@dmitryax PTAL

File: distributions/otelcol-contrib/manifest.yaml [hunk 1]
```diff
@@ -140,6 +141,7 @@ receivers:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/podmanreceiver v0.58.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/postgresqlreceiver v0.58.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.58.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/receiver/pulsarreceiver v0.0.0-00010101000000-000000000000
```
Comment (dmitryax) [inline distributions/otelcol-contrib/manifest.yaml:144]: The receiver cannot be added until the contrib go library is released

Comment (tjiuming) [reply distributions/otelcol-contrib/manifest.yaml]: @dmitryax resolved.

File: distributions/otelcol-contrib/manifest.yaml [hunk 2]
```diff
@@ -166,4 +167,4 @@ replaces:
   # See https://github.com/google/gnostic/issues/262
   - github.com/googleapis/gnostic v0.5.6 => github.com/googleapis/gnostic v0.5.5
   # See https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12322#issuecomment-1185029670
-  - github.com/docker/go-connections v0.4.1-0.20210727194412-58542c764a11 => github.com/docker/go-connections v0.4.0
\ No newline at end of file
+  - github.com/docker/go-connections v0.4.1-0.20210727194412-58542c764a11 => github.com/docker/go-connections v0.4.0
```
Comment (dmitryax) [inline distributions/otelcol-contrib/manifest.yaml:170]: Please avoid unrelated/accidental changes

Comment (tjiuming) [reply distributions/otelcol-contrib/manifest.yaml]: sorry, goland's issue. I've resolved

**Review by dmitryax — APPROVED** [review]

**Pull request merged**